src/tests/unit/doughs-serializer.test.js

```diff
@@ -0,0 +1,120 @@
+import chai from 'chai';
+import config from 'config';
+import _ from 'lodash';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+
+import { openapi } from 'utils/load-openapi';
+import { doughSerializerData } from './test-data';
+
+const doughAttributes = openapi.definitions.DoughAttributes.properties;
+
+chai.should();
+
+let rawDough;
+let rawDoughs;
+let doughSerializer;
+
+const {
+  nullSpecialInstructionsDough,
+  emptyStringSpecialInstructionsDough,
+} = doughSerializerData;
+
+describe('test doughs serializer', () => {
+  before(() => {
+    sinon.replace(config, 'get', () => ({ oracledb: {} }));
+    // importing doughSerializer normally causes config to become unextensible,
+    // which then breaks other tests. This prevents that problem, even though
+    // stubbing imports is not required.
+    doughSerializer = proxyquire('../../api/v1/serializers/doughs-serializer', {});
+  });
+  after(() => {
+    sinon.restore();
+  });
+  describe('serializeDough', () => {
+    describe('when it gets a dough object with specialInstructions = `null`', () => {
+      beforeEach(() => {
+        rawDough = nullSpecialInstructionsDough;
+      });
+      it('converts specialInstructions to an empty string', () => {
```

Comment:
This test case and the following one are almost the same. Make use of loops to reduce code reuse. Here is an example in my [learning project](https://github.com/osu-mist/game-curator-api/blob/4929a92be94da94b9fb2084382360c9832da033b/tests/unit/games-dao.test.js#L112-L133).